TerminalDocs/customize-settings/profile-appearance.md

```diff
@@ -399,179 +267,110 @@ When `useAcrylic` is set to `true`, this sets the transparency of the window for
-When this is set to `true`, the window will scroll to the command input line when typing. When it's set to `false`, the window will not scroll when you start typing.
+This changes the foreground color of the profile. This overrides `foreground` set in the color scheme if `colorScheme` is set.
 
-**Property name:** `snapOnInput`
+**Property name:** `foreground`
 
 **Necessity:** Optional
 
-**Accepts:** `true`, `false`
-
-**Default value:** `true`
+**Accepts:** Color as a string in hex format: `"#rgb"` or `"#rrggbb"`
 
-### History size
+### Background color
 
-This sets the number of lines above the ones displayed in the window you can scroll back to.
+This changes the background color of the profile with this setting. This overrides `background` set in the color scheme if `colorScheme` is set.
 
-**Property name:** `historySize`
+**Property name:** `background`
 
 **Necessity:** Optional
 
-**Accepts:** Integer
-
-**Default value:** `9001`
-
-<br />
-
-___
+**Accepts:** Color as a string in hex format: `"#rgb"` or `"#rrggbb"`
 
-## How the profile closes when exiting
+### Selection background color
 
-This sets how the profile reacts to termination or failure to launch. `"graceful"` will close the profile when `exit` is typed or when the process exits normally. `"always"` will always close the profile and `"never"` will never close the profile. `true` and `false` are accepted as synonyms for `"graceful"` and `"never"`, respectively.
+This sets the background color of a selection within the profile. This will override the `selectionBackground` set in the color scheme if `colorScheme` is set.
 
-**Property name:** `closeOnExit`
+**Property name:** `selectionBackground`
 
 **Necessity:** Optional
 
-**Accepts:** `"graceful"`, `"always"`, `"never"`, `true`, `false`
-
-**Default value:** `"graceful"`
-
-<br />
-
-___
+**Accepts:** Color as a string in hex format: `"#rgb"` or `"#rrggbb"`
 
-## Bell settings ([Preview](https://aka.ms/terminal-preview))
+### Cursor color
 
-Controls what happens when the application emits a BEL character. When set to `"audible"`, the terminal will play a sound. When set to `"none"`, nothing will happen.
+This sets the cursor color of the profile. This will override the `cursorColor` set in the color scheme if `colorScheme` is set.
 
-**Property name:** `bellStyle`
+**Property name:** `cursorColor`
 
 **Necessity:** Optional
 
-**Accepts:** `"audible"`, `"none"`
-
-**Default value:** `"audible"`
-
-> [!IMPORTANT]
-> This feature is only available in [Windows Terminal Preview](https://aka.ms/terminal-preview/).
+**Accepts:** Color as a string in hex format: `"#rgb"` or `"#rrggbb"`
 
 <br />
 
 ___
 
-## Retro terminal effects
-
-:::row:::
-:::column span="":::
-When this is set to `true`, the terminal will emulate a classic CRT display with scan lines and blurry text edges. This is an experimental feature and its continued existence is not guaranteed.
+## Pixel shader effects ([Preview](https://aka.ms/terminal-preview))
```

Comment:
Add a note that this isn't in SUI?

Reply:
Also, maybe this should be moved up by retro terminal effects?

Reply:
It'd be nice if there was a tutorial for this, but that's probably more of a feature request than anything.

Reply:
Pushing this off until the settings UI becomes default.

I put all of the settings that aren't in the UI at the bottom of the page, which is why this isn't near retro terminal effects.